app/src/components/main/analytics/events/common/testItemPages/modalEventsCreators.js

```diff
@@ -224,6 +224,11 @@ export const getEditItemsModalEvents = (category, itemType = 'Item') => ({
     action: `Edit description in Modal "Edit ${itemType}"`,
     label: 'Edit description',
   },
+  CLICK_COPY_LINK: {
+    category,
+    action: `Click on Icon Copy`,
+    label: 'Modal Edit Launch',
```

Comment:
Should it be called `Modal Edit Item` when the event triggered from suites/tests/steps?